packages/build-tools/src/utils/project.ts

```diff
@@ -137,15 +135,48 @@ export async function isUsingYarn2(projectDir: string): Promise<boolean> {
   return (await fs.pathExists(yarnrcPath)) || (await fs.pathExists(yarnrcRootPath));
 }
 
+export function runExpoCliCommand<TJob extends Job>(
+  ctx: BuildContext<TJob>,
+  args: string[],
+  options: SpawnOptions,
+  { forceUseGlobalExpoCli = false } = {}
+): SpawnPromise<SpawnResult> {
+  if (
+    !forceUseGlobalExpoCli &&
+    ctx.env.EXPO_USE_LOCAL_CLI !== '0' &&
```

Comment:
this is a weird way of configuring it, if you want a way to disable local cli when it should be used env should be called DISABLE_LOCAL_CLI or EXPO_USE_GLOBAL_CLI.

The current setup is counterintuitive because setting getPrebuildCommand=1 does not guarantee that local cli will be used